### ISSUE TITLE
[FF33 TU1.5.28.1] Tab stacking: Drag to stack doesn't work

### ISSUE THREAD

**Issue by RedHorizon** [ithinc/tabutils#132]
FF33, TU1.5.28.1
Create a new FF profile, launch it and install TU
extensions.tabutils.autoStack true, extensions.tabutils.dragToStack true
Open 2 tabs (2 blank tabs is ok), drag right tab on top of the left one - only right tab will be colored and have Tab stack option in rmb menu, left tab is not marked as tab in stack and not colored

**Comment by mce128** [comment]
Yes, I am experiencing this issue as well. In my extant config and in new profiles created for testing. Also, I'd just add that while 2 blank tabes are ok to reproduce, any set of tabs you try this with will exhibit this problem. It also doesn't matter how many or how few tabs you have overall (aside from requiring a minimum of two to duplicate, obviously.)
